sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1311,6 +1311,10 @@ export class ApplicationService extends RecordDataModelService<Application> {
         id: true,
         applicationNumber: true,
         isArchived: true,
+        data: {
```

Comment:
Unfortunately, Typeorm does not support this type of query and this will not produce the expected result. The entire payload will be retrieved either way. To avoid this we need special JSONB instructions that we can use if there is a performance concern. For this case, the below suggestion would be preferred.
Can you please change it as below:
```ts
data: true as unknown
```

Reply:
if it would have be returned like that, that would be of great use :)